frontend/src/features/the-erfahre-mehr/common/components/BackButton.vue

```diff
@@ -0,0 +1,44 @@
+<template>
```

Comment:
Finde ich gut. Ich würde das aber etwas "höher" in der Struktur packen, da wir das auch auf den anderen Seiten Nutzen können.